CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 - Updates how styles are passed down to internal components in `Card`, `Checkbox`, `CheckboxGroup`, `ComponentWrapper`, `DatePicker`, `Hero`, `Notification`, `Pagination`, `Radio`, `RadioGroup`, `SearchBar`, and `Slider`. This is based on removing the `additionalStyles` prop and passing down styles to the `__css` prop.
 - Updates how the `id` is passed in the `DatePicker`'s custom `TextInput` component.
+- Added `noSpace` prop to the `Heading` component.
```

Comment:
This should go under a new heading `### Adds` above the `### Updates` heading. Also, "Added" should be updated to "Adds" to keep the same tense.